### PR TITLE
get MQTT endpoint from API instead of using a hardcoded endpoint

### PR DESCRIPTION
**Pull request by Bouni**
This change should prevent possible future changes to the API endpoint

### CODE REVIEW

File: landroid-cloud.js [hunk 1]
```diff
@@ -165,7 +170,7 @@ LandroidCloud.prototype.connectToMQTT = function () {
         console.log("Connecting to MQTT broker");
```
Comment (mjiderhamn) [inline landroid-cloud.js]: May I suggest including the host name in this log message?

File: landroid-cloud.js [hunk 2]
```diff
@@ -62,6 +62,11 @@ LandroidCloud.prototype.setToken = function (token) {
   console.log("API token set to " + this.token);
 };
 
+LandroidCloud.prototype.setMQTTendpoint = function (endpoint) {
```
Comment (mjiderhamn) [inline landroid-cloud.js]: Please use uppercase E for consistency (`setMQTTEndpoint`)

File: landroid-cloud.js [hunk 3]
```diff
@@ -62,6 +62,11 @@ LandroidCloud.prototype.setToken = function (token) {
...
+  this.mqtt_endpoint = endpoint;
```
Comment (mjiderhamn) [inline landroid-cloud.js]: Please use camel case variable names rather than underscore, for consistency

File: landroid-cloud.js (hunk 1 above)
Comment (Bouni) [reply landroid-cloud.js]: Will do that!

File: landroid-cloud.js (hunk 2 above)
Comment (Bouni) [reply landroid-cloud.js]: My bad, will be fixed as well

File: landroid-cloud.js (hunk 3 above)
Comment (Bouni) [reply landroid-cloud.js]: Ok, I just took the variable name from the json.

**Comment by mjiderhamn** [comment]
Thanks for the Pull Request!

**Pull request merged**